qiskit/visualization/circuit/latex.py

```diff
@@ -183,15 +184,23 @@ def __init__(
             self._layout = None
 
         self._initial_state = initial_state
-        self._cregbundle = cregbundle
         self._global_phase = circuit.global_phase
 
         # If there is any custom instruction that uses classical bits
         # then cregbundle is forced to be False.
-        for layer in self._nodes:
-            for node in layer:
-                if node.op.name not in {"measure"} and node.cargs:
-                    self._cregbundle = False
+        for node in itertools.chain.from_iterable(self._nodes):
+            if node.cargs and node.op.name != "measure":
```

Comment:
I know this is pre-existing but doesn't this potentially miss a op with a condition too?

Reply:
Conditions have special handling in `cregbundle=True` format, since they're limited to acting on at most a single bit of the register.  We can operate on a circuit with single-bit conditions even in that mode, which I think is the reason it's skipped.